specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/examples/SuccessfulAnalyzeConversationTextJobsTaskSubmitRequest.json

```diff
@@ -0,0 +1,58 @@
+{
+  "parameters": {
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "api-version": "2022-04-01-preview",
+    "Endpoint": "{Endpoint}",
+    "jobId": "{Job ID}",
+    "body": {
+      "displayName": "Analyze conversations from xxx",
+      "analysisInput": {
+        "conversations": [
+          {
+            "id": "20220101meeting",
+            "language": "en",
+            "modality": "text",
+            "domain": "neutral",
+            "conversationItems": [
+              {
+                "participantId": "user",
+                "id": "1",
+                "modality": "text",
+                "text": "Hi!"
+              },
+              {
+                "participantId": "agent",
+                "id": "2",
+                "modality": "text",
+                "text": "Hello, how can I help you?"
+              },
+              {
+                "participantId": "user",
+                "id": "3",
+                "modality": "text",
+                "text": "I am having trouble issuing a return of a game on my xbox: call of duty"
+              }
+            ]
+          }
+        ],
+        "tasks": [
+          {
+            "taskName": "analyze 1",
+            "kind": "IssueResolutionSummarization",
+            "parameters": {
+              "modelVersion": "2022-04-01",
+              "summaryAspects ": "issue, resolution"
```

Comment:
It would be good to follow the same principal as `piiCategories` here in regards to defaults and "ALL" value.

Reply:
Adding @mikaelsitruk can you comment.